risedev.yml

```diff
@@ -703,12 +704,21 @@ template:
     # If `user-managed` is true, this service will be started by user with the above config
     user-managed: false
 
+  # Whether to enable in-memory pure KV state backend. When set to true, no compactor should be used.
+    enable-in-memory-kv-state-backend: false
```

Comment:
Should we remove the configuration with the same name for compute nodes? 🤔

Reply:
Risedev still needs this option to derive `state-store` command line argument. Since it is kept in this PR, maybe we should remove it in the next PR?